release/models/system/openconfig-system-logging.yang

```diff
@@ -473,6 +479,142 @@ module openconfig-system-logging {
     }
   }
 
+  grouping logging-file-config {
+    description
+      "Configuration data for logfile";
+
+    leaf filename-prefix {
+      type string {
+        length 0..255;
+      }
+      description
+        "filename prefix. It is expected that implementation may append to it timestamp, serial-number or other type of identifier";
+    }
+
+    leaf path {
+      type string {
+        length 0..255;
+      }
+      description
+        "path to the folder where logfile is stored. May include drive identifier is implementation requires";
+    }
+
+    leaf count {
+      type uint32;
+      default 1;
+      description
+        "Used for logfile rotation. maximum number of files of given filename-prefix stored. Default value: 1. ";
```

Comment:
Need to clarify what happens is there is only 1 or 0 files.  I suggested a definition following linux log rotate behavior.  But this should be compared with network device implementations.  Note, defaults do not have to match, but the behavior should be similar enough that the OC can be translated to a behavior that at least 2 vendors implement.

```suggestion
        "Used for logfile rotation.   When the max-size limit is reached the active file is rotated.  Log files are rotated the number of times defined by this leaf.  The default value of 1 indicates that there will be one rotation file and one active file.  A 0 value indicates old versions are removed rather than rotated.";
```